modules/ROOT/pages/inline-css.adoc

```diff
@@ -0,0 +1,116 @@
+= Inline CSS plugin
+:navtitle: Inline CSS
+:description: takes {productname} documents and outputs HTML formatted according to email-friendly HTML best practices (eg inlined CSS).
+:description_short: Inlines {productname} CSS for better presentation in email.
+:keywords: plugin, inlinecss, email friendly, email, output, inline, css, post-process, html
+:pluginname: Inline CSS
+:plugincode: inlinecss
+:plugincategory: premium
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.3v.adoc[]
+
+As is normal and long-time best practice, {productname} documents keep CSS and HTML separate. This practice is not, however, email-friendly.
+
+The {pluginname} plugin takes a {productname} document, and processes it such that the previously separate CSS is applied inline to each HTML element. This single output file is much closer to the common requirements for sending as an HTML-formatted email.
+
+== Interactive example
+
+////
+  No Lorem Ipsum! Add some content to test on, possibly just a description of how end-users use the feature?
+  See the emoticons, advtable,and advcode demos for examples.
+////
+
+liveDemo::inline-css[]
+
+== Basic setup
+
+To add the {pluginname} plugin to the editor, add `{plugincode}` to the `plugins` option in the editor configuration.
+
+For example:
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: '{plugincode}',
```

Comment:
This part still show up as raw `{pluginname}` in the staging build, might need to change to hard coded `inlinecss`?

Reply:
```suggestion
  plugins: 'inlinecss',
```